docs/CONTRIBUTING.md

```diff
@@ -5,60 +5,129 @@ route: /CONTRIBUTING.md
 
 # Contributing
 
-# How can you contribute?
+## How can you contribute?
 
 There are several ways you can make meaningful contributions to `Atlantis`.
 
-## Submit an Issue
+### Types of contribution
 
-Submitting an issue is probably the easiest way you can contribute to
-`Atlantis`.
+#### Bugfix
+
+If a component’s layout is broken, fails a WCAG standard, or blows up when used
+as intended, there’s no need to discuss fixing it - put up a PR!
+
+#### Feature enhancement
+
+If a component does almost the thing you need, or looks almost right for your
+use case, but you need to extend it a bit to get it to work just right, talk to
+at least one other designer. The outcome of this talk should be alignment on the
```

Comment:
This jumps out to me. We are branding 🔱 as an open source project. If someone who doesn't work at Jobber (say their name is Daryl) wants to contribute, how do they reach a designer?

Reply:
Good call! Updated the "talk to {Jobber person or team}" references to be less specific and also allow for putting up an issue.